packit.yaml

```diff
@@ -21,18 +21,13 @@ jobs:
     targets:
     - fedora-development
 
+  # for cross-project testing
   - job: copr_build
-    trigger: release
+    trigger: commit
+    branch: "^main$"
```

Comment:
I think we can omit this which will use default branch.

Honestly, I'm not sure if regular expressions works here...